flow-nodes/api-builder-plugin-typeof/.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+package-lock.json
```

Comment:
Be sure to add a newline to the end of each file

Reply:
New line added.